File: password.java
import java.util.Random;
import java.util.Scanner;
public class password {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Random ran = new Random();

        System.out.println("Enter the Password Length : ");
        int len = sc.nextInt();
        
        System.out.print("Password is : ");
        for(int i=0;i<len;i++){
            System.out.println(ran.nextInt(10));
        }
    }
}

File: password.js
const pass_el = document.querySelector('#password span');
const len = 8;
const chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!@#$%&*(){}[]/+_-"

function genPassword(){
    let pass = "";
    for(var i=0;i<=length;i++){
        let ran = Math.floor(Math.random()*chars.length);
        pass += chars.substring(ran,ran+1);
    }
}